src/scikitlearn.jl

```diff
@@ -68,7 +68,7 @@ end
 
 # The input matrix is called X (instead of A) following ScikitLearn's convention
 function ScikitLearnBase.fit_transform!(skglrm::SkGLRM, X, y=nothing;
-                                        missing_values::Matrix=isnan(X))
+                                        missing_values=isnan.(X))
```

Comment:
Did you try it? It works on my machine, on 0.5.1. It's [loop fusion](http://julialang.org/blog/2017/01/moredots). Without the dot there's a warning on 0.6.